opentelemetry/src/sdk/trace/span_processor.rs

```diff
@@ -125,13 +130,17 @@ impl SpanProcessor for SimpleSpanProcessor {
         }
     }
 
-    fn force_flush(&self) {
+    fn force_flush(&self) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
         // Ignored since all spans in Simple Processor will be exported as they ended.
+        Ok(())
     }
 
-    fn shutdown(&mut self) {
+    fn shutdown(&mut self) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
         if let Ok(mut exporter) = self.exporter.lock() {
             exporter.shutdown();
+            Ok(())
+        } else {
+            Err(TraceError::Other("When force flushing the SimpleSpanProcessor, the exporter's lock has been poisoned".into()).into())
```

Comment:
should this comment say when shutting down rather than force flushing?

Reply:
Yeah good catch!